lib/views/client-view-mgr.js

```diff
@@ -4,14 +4,14 @@ var Promise = require('bluebird');
 var Base = require('extendable-base');
 var JuttleErrors = require('../errors');
 
-var ClientSinkManager = Base.extend({
+var ClientViewManager = Base.extend({
```

Comment:
Sure, I'll rename (and also rename lib/views/client-view-mgr.js to lib/views/view-mgr.js).